src/main/resources/db/changelog/v.0.4/db.changelog-master.yaml

```diff
@@ -1,13 +1,22 @@
 databaseChangeLog:
- - include:
-      file: db.changelog-user-internship.yaml
-      relativeToChangelogFile: true
  - include:
       file: db.changelog-change-size-of-password.yaml
       relativeToChangelogFile: true
  - include:
       file: db.changelog-create-table-country-and-city.yaml
       relativeToChangelogFile: true
+ - include:
```

Comment:
why did you move already executed changelog?

Reply:
As far as I remember during resolving conflicrs I made a mistake and clicked "x" insted of "<<<" and then I added to the end, I'll fix it to avoid futher stakeholders conflicts

Reply:
Fix please in `application.yaml ` also

Reply:
Done